package.json

```diff
@@ -18,7 +18,7 @@
     "ast-types": "git+https://github.com/jlongster/ast-types.git",
     "babylon": "^6.15.0",
     "esutils": "^2.0.2",
-    "flow-parser": "^0.37.0",
+    "flow-parser": "0.38.0",
```

Comment:
I actually like it, but it's up to you. Before yarn I never used `^` so I'm used to avoiding it. Since we're using yarn I think it makes sense to avoid it.